File: dsDetail.vb
Partial Class dsDetail
    Partial Class spPlanEstimatesGetDataTable

        Private Sub spPlanEstimatesGetDataTable_ColumnChanging(sender As Object, e As DataColumnChangeEventArgs) Handles Me.ColumnChanging
            If (e.Column.ColumnName = Me.ActivityCodeColumn.ColumnName) Then
                'Add user code here
            End If

        End Sub

        Private Sub spPlanEstimatesGetDataTable_RowChanged(sender As Object, e As DataRowChangeEventArgs) Handles Me.RowChanged
            ''            e.Row.Item("UpdatedBy") = gsUserID
            ' Me. = e.Row.Item("PlanEstimateID")
        End Sub
    End Class

End Class

Namespace dsDetailTableAdapters
    
    Partial Public Class spPlanEstimatesGetTableAdapter

    End Class
End Namespace

File: dsDetail.cs
using System.Data;

namespace BossAdmin
{
    public partial class dsDetail
    {
        public partial class spPlanEstimatesGetDataTable
        {

            private void spPlanEstimatesGetDataTable_ColumnChanging(object sender, DataColumnChangeEventArgs e)
            {
                if ((e.Column.ColumnName??"")==(ActivityCodeColumn.ColumnName??""))
                {
                    // Add user code here
                }

            }

            private void spPlanEstimatesGetDataTable_RowChanged(object sender, DataRowChangeEventArgs e)
            {
                // '            e.Row.Item("UpdatedBy") = gsUserID
                // Me. = e.Row.Item("PlanEstimateID")
            }
        }

    }

    namespace dsDetailTableAdapters
    {

        public partial class spPlanEstimatesGetTableAdapter
        {

        }
    }
}